### ISSUE TITLE
Get rid of cookies on embedded iframes

### ISSUE THREAD

**Issue by kookster** [PRX/theworld.org#446]
we are registering 3rd party cookies on blacklight now, to spotify, I believe b/c of this iframe on this page https://theworld.org/categories/planet-hip-hop

This can be resolved by using the sandbox attribute on the iframe  <iframe sandbox="allow-scripts"
e.g.
```
    <iframe sandbox="allow-scripts"
        allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture" allowfullscreen=""
        frameborder="0" height="352" loading="lazy"
        src="https://open.spotify.com/embed/playlist/3QaoOet1YYTp7grDyuT0S6?utm_source=generator" width="100%">
    </iframe>
```
also https://github.com/itteco/iframely/issues/405

oh interesting, and for youtube, we can set up privacy enhancing mode - https://axbom.com/embed-youtube-videos-without-cookies/

basically you switch [youtube.com](http://youtube.com/) to [youtube-nocookie.com](http://youtube-nocookie.com/)
oh that still calls local storage, ugh, well, better at least

**Comment by rpeterman-gp** [comment]
This was resolved by #465 and PRX/cms.theworld.org#217
